tardis/montecarlo/base.py

```diff
@@ -156,7 +156,6 @@ def __init__(
     def _initialize_estimator_arrays(self, tau_sobolev_shape):
         """
         Initialize the output arrays of the montecarlo simulation.
-
```

Comment:
This whitespace should not be removed from docstrings.

Reply:
Sure! Resolved this.

Reply:
This applies to every docstring where the whitespace was applied

Reply:
@andrewfullard Yeah sorry my bad. Fixed them all. (I hadn't explicitly made these changes, my Black formatted did them on save. I have disabled my Black formatter now)